rudder-agent/SOURCES/rudder-agent.init

```diff
@@ -191,6 +191,17 @@ start_daemons() {
 
 	# It's time to start the daemons
 	for daemon in ${DAEMONS}; do
+    # Test if daemon is already running
+    if [ -s ${CFENGINE_COMMUNITY_PID_FILE[$daemon]} ]
+    then
+      PID=`cat ${CFENGINE_COMMUNITY_PID_FILE[$daemon]}`
+      if ps -p "${PID}" > /dev/null
```

Comment:
PS_COMMAND is for process that have gone wild and have no pid file, it is used to detect and kill them.
Regular process run by the service should have proper pid file, the one we check here.